test/SILGen/opaque_values_silgen.swift

```diff
@@ -8,6 +8,19 @@ protocol P {
   var x : Int { get }
 }
 
+func hasVarArg(_ args: Any...) {}
+
+// Test that we still use addresses when dealing with array initialization
+// ---
+// CHECK-LABEL: sil @_TF20opaque_values_silgen10callVarArgFT_T_ : $@convention(thin) () -> () {
+// CHECK: apply %{{.*}}<Any>(%{{.*}}) : $@convention(thin) <τ_0_0> (Builtin.Word) -> (@owned Array<τ_0_0>, Builtin.RawPointer)
+// CHECK: tuple_extract %{{.*}} : $(Array<Any>, Builtin.RawPointer)
+// CHECK: pointer_to_address %{{.*}} : $Builtin.RawPointer to [strict] $*Any
+// CHECK: init_existential_addr %{{.*}} : $*Any, $Int
```

Comment:
Can you add more precise FileCheck lines here? I prefer to use register matching like:

[[FOO:%.*]]

Also can you use a } // end sil function '' check to make sure that we do not match lines below?

I have run into many of these issues in SILGen = /.